os_net_config/impl_ifcfg.py

```diff
@@ -259,43 +259,42 @@ def ifcfg_requires_restart(self, filename, new_data):
         """
 
         file_data = common.get_file_data(filename)
-        logger.debug("Original ifcfg file:\n%s" % file_data)
-        logger.debug("New ifcfg file:\n%s" % new_data)
+        logger.debug(f'Original ifcfg file:\n{file_data}')
+        logger.debug(f'New ifcfg file:\n{new_data}')
         file_values = self.parse_ifcfg(file_data)
         new_values = self.parse_ifcfg(new_data)
         restart_required = False
         # Certain changes can be applied without restarting the interface
         permitted_changes = [
-            "IPADDR",
-            "NETMASK",
-            "MTU",
-            "ONBOOT",
-            "ETHTOOL_OPTS"
+            'IPADDR', 'NETMASK',
+            'MTU', 'ONBOOT', 'ETHTOOL_OPTS',
+            'DOMAIN', 'DNS1', 'DNS2'
```

Comment:
If IPADDR is supported, shouldn't "IPV6ADDR" also be allowed without reboot ?

Reply:
Maybe in a follow up